### ISSUE TITLE
Octokit.Reactive PCL support

### ISSUE THREAD

**Issue by shiftkey** [octokit/octokit.net#435]
Once #401 lands, we should also look at doing a PCL for Octokit.Reactive

**Comment by shiftkey** [comment]
Dropping this from the VS2015 milestone - we're still waiting on upstream support for DNX Core from Rx.NET

**Comment by shiftkey** [comment]
PCLs are dead, DNX is dead, long live .NET Standard (we target .NET Standard 1.1 these days for both libraries).

Someone else can open a new issue about this if there's a chance to improve things.
